lib/archiving.rb

```diff
@@ -27,11 +27,16 @@ def self.archive(date)
           end
         end
       end
+
       puts "Removing archived data from database for #{date}..."
       deliveries.find_each do |delivery|
         delivery.destroy
       end
+
       copy_to_s3(date)
+
+      puts "Removing temp local file #{date}.tar.gz copied to S3..."
+      File.delete("db/archive/#{date}.tar.gz")
```

Comment:
I've now changed this so that it if S3 isn't configured, or there's an error raised when it tries to upload, it won't delete the file and tells you what it's doing on the console output.

There's tests covering this here: https://github.com/mlandauer/cuttlefish/pull/270/files#diff-01d06a85c92ee1298228bf80d6880c96R49